### PR TITLE
Fix sample usage mentioned in #1168

### PR DESCRIPTION
**Pull request by alexeyzimarev**

### CODE REVIEW

File: src/app/FakeLib/PaketTemplateHelper.fs [hunk 1]
```diff
@@ -254,21 +254,23 @@ module internal Rendering =
...
-///                                     Authors = ["Me"; "You"; "Someone Other"] })
-///                                     Files = [ Include ("./**/*.dll", "/lib/dlls")
-///                                               Exclude "./Foo/bar.dll"
-///                                               Include ("./*.jpg", "/images") ]
-///                                     Dependencies = [ "Paket1.FOO", AnyVersion
-///                                                      "Paket2.BAR", GreaterOrEqual CURRENTVERSION
-///                                                      "Paket3.BAZ", GreaterOrEqualSafe LOCKEDVERSION
-///                                                      "Paket4.BOO", GreaterOrEqual (Version "1.2.3") ]
-///     )
+///Target "Create Paket Template" (fun _ ->
```
Comment (forki) [inline src/app/FakeLib/PaketTemplateHelper.fs]: I don't think the indentation is right here

Comment (alexeyzimarev) [inline src/app/FakeLib/PaketTemplateHelper.fs]: I pasted from a working script

Comment (forki) [inline src/app/FakeLib/PaketTemplateHelper.fs]: yes I know, but take a look at the diff.
Inorder to get properly highlighted as code you need to indent 4 spaces

**Comment by alexeyzimarev** [comment]
I see, I think it is OK now

**Comment by forki** [comment]
thx

**Pull request merged**